benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -19,57 +19,57 @@ msgstr "Get started with"
 msgid "eligibility.buttons.signout"
 msgstr "Sign out of Login.gov"
 
-msgid "eligibility.pages.index.mst_login.label"
+msgid "eligibility.pages.index.login_gov.label"
 msgstr "I am 65 years of age or older"
 
-msgid "eligibility.pages.index.mst_login.description"
+msgid "eligibility.pages.index.login_gov.description"
 msgstr ""
 "You must be 65 years or older. This benefit does not expire, but you may "
 "need to renew. Using this benefit means your new transit fare is half of the "
 "standard fare."
 
-msgid "eligibility.pages.start.mst_login.title"
+msgid "eligibility.pages.start.login_gov.title"
 msgstr "Older Adult benefit information"
 
-msgid "eligibility.pages.start.mst_login.headline"
+msgid "eligibility.pages.start.login_gov.headline"
 msgstr "You selected an Older Adult benefit."
 
-msgid "eligibility.pages.start.mst_login.sub_headline"
+msgid "eligibility.pages.start.login_gov.sub_headline"
 msgstr "You will need a few items to connect your benefit:"
 
-msgid "eligibility.pages.start.mst_login.start_item.heading"
+msgid "eligibility.pages.start.login_gov.start_item.heading"
 msgstr "A Login.gov account with identity verification"
 
-msgid "eligibility.pages.start.mst_login.start_item.details"
+msgid "eligibility.pages.start.login_gov.start_item.details"
 msgstr ""
 "Login.gov is a safe way to sign in to government services. Benefits uses "
 "Login.gov to verify your age. If you do not have an account already, you "
 "will be able to create one. You will also need to verify your identity, "
 "which will require these items:"
 
-msgid "eligibility.pages.unverified.mst_login.title"
+msgid "eligibility.pages.unverified.login_gov.title"
 msgstr "Eligibility Error"
 
-msgid "eligibility.pages.unverified.mst_login.headline"
+msgid "eligibility.pages.unverified.login_gov.headline"
 msgstr "Your eligibility could not be verified."
 
-msgid "eligibility.pages.unverified.mst_login.p[0]"
+msgid "eligibility.pages.unverified.login_gov.p[0]"
 msgstr ""
 "That’s okay! You may still be eligible for our program. Please reach out to "
 "your local transit provider for assistance."
 
-msgid "enrollment.pages.index.mst_login.eligibility_confirmed.headline"
+msgid "enrollment.pages.index.login_gov.eligibility_confirmed.headline"
 msgstr "Your eligibility is confirmed! You’re nearly there."
 
-msgid "enrollment.pages.index.mst_login.eligibility_confirmed_item.heading"
+msgid "enrollment.pages.index.login_gov.eligibility_confirmed_item.heading"
 msgstr "Your eligibility is confirmed for an Older Adult benefit"
 
 msgid "enrollment.pages.index.mst_login.eligibility_confirmed_item.details"
 msgstr ""
 "Your eligibility has been verified by Login.gov. This benefit will give you "
 "a half-price fare on public transportation with MST."
```

Comment:
Leaving `enrollment.pages.index.mst_login.eligibility_confirmed_item.details` alone to be handled by #1187 since it has the "with MST" part in it.

Similarly, also leaving `core.pages.agency_index.mst_cc.headline` alone to be handled by #1187 -- it is used in the `agency_index` view function, which applies to all agencies.

https://github.com/cal-itp/benefits/blob/22cfc1677349d175e4875f044be17482aa52b534/benefits/locale/en/LC_MESSAGES/django.po#L425-L426

https://github.com/cal-itp/benefits/blob/22cfc1677349d175e4875f044be17482aa52b534/benefits/core/views.py#L61